resources/authgear/templates/en/web/authflowv2/bot_protection_widget.html

```diff
@@ -0,0 +1,16 @@
+{{ $cloudflareTurnstileWidgetTheme := "light" }}
+{{ $recaptchav2WidgetTheme := "light" }}
+{{ if $.DarkThemeEnabled }}
+ {{ $cloudflareTurnstileWidgetTheme = "dark" }}
+ {{ $recaptchav2WidgetTheme = "dark" }}
+{{ end }}
+
+{{ if $.BotProtectionRequired }}
+  <input type="hidden" name="x_bot_protection_provider_type" value="{{$.BotProtectionProviderType}}">
+  <input type="hidden" name="x_bot_protection_provider_response">
+  {{ if (eq $.BotProtectionProviderType "cloudflare" ) }}
+    <div id="cloudflare-turnstile-widget" class="flex justify-center" data-controller="cloudflare-turnstile" data-cloudflare-turnstile-site-key-value="{{$.BotProtectionProviderSiteKey}}" data-cloudflare-turnstile-theme-value="{{$cloudflareTurnstileWidgetTheme}}"></div>
+  {{ else if (eq $.BotProtectionProviderType "recaptchav2" ) }}
+    <div id="recaptchav2-widget" class="flex justify-center" data-controller="recaptcha-v2" data-recaptcha-v2-site-key-value="{{$.BotProtectionProviderSiteKey}}" data-recaptcha-v2-theme-value="{{$recaptchav2WidgetTheme}}"></div>
```

Comment:
The mapping of theme should happen in the controller. The controller handles the integration between Authgear and the provider script API.

And you should read `<meta name=x-color-scheme>` to get the current color scheme. `$.DarkThemeEnabled` merely tells dark theme is enabled. It does not mean it is dark mode.

Since AvailableLanguages are defined in the server side, we need to add a comment there to tell the future editor to update the mapping when a new available language is added.